tests/unit/test_cromshellconfig.py

```diff
@@ -0,0 +1,97 @@
+from cromshell.utilities import cromshellconfig
+from importlib import reload
+from pathlib import Path
+import pytest
+import os
+
+
+class TestCromshellConfig:
+    """Test the cromshell config functions and variables"""
+
+    def test_override_slim_metadata_parameters(self):
+        reload(cromshellconfig)
+        default_slim_parameters = (
```

Comment:
Avoid redefining default values in tests. Instead read them from the place where they are originally defined (in this case, `cromshellconfig`). This avoids headaches of having to manually sync the two definitions.